src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -1,7 +1,8 @@
 package core.basesyntax.service;
 
+import core.basesyntax.exceptions.RegistrationException;
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws RegistrationException;
```

Comment:
should we indicate that method throws RuntimeException?

Reply:
Yeap, you're right. In my first version, I made a checked exception and after changing I didn't fix it